src/Dev/Deprecation.php

```diff
@@ -73,11 +79,20 @@ class Deprecation
      * @var int - the notice level to raise on a deprecation notice. Defaults to E_USER_DEPRECATED if that exists,
      * E_USER_NOTICE if not
      */
-    public static $notice_level = null;
+    public static $notice_level = E_USER_DEPRECATED;
```

Comment:
I don't think we should keep this - `E_USER_DEPRECATED` has been around since PHP 5.3 (https://wiki.php.net/rfc/e-user-deprecated-warning) and from the comment here it seems the only reason this was created was in case that constant isn't available.

Reply:
This public property is changed in DeprecationTest to E_USER_NOTICE to raise the level so that `$this->expectError()` works.  The comment above this seems out of date so I'll just remove it

Reply:
Have deprecated the property

Reply:
We need to retain this for unit-testing to swap it out for `E_USER_ERROR` - reason being that `expectedDeprecated()` does not work with `E_USER_DEPRECATED` - https://github.com/laminas/laminas-di/pull/30#issuecomment-927585210